docs/_tutorials/scenarios_suites/HuggingFace.rst

```diff
@@ -0,0 +1,70 @@
+HuggingFace's NLP datasets
+----------------------------
+
+
+HuggingFace proposes the largest amount of NLP datasets. The full list
+can be found `here <https://huggingface.co/docs/datasets/>__`.
+
+Naturally, Continuum can work with those NLP datasets and convert them in
+continual scenarios.
+
+In all cases, you won't work with `TaskSet` but instead directly with HuggingFace's Dataset.
+
+HuggingFace Continual
+=========================
+
+`HuggingFaceIncremental` can allow you to split a HuggingFace dataset according
+a to field name. For example, in the next code block, I download the MultiNLI dataset
+with HuggingFace. Then, I'm asking to split this dataset according to the
+`genre` field, and I'm also asking to see only 1 genre per task (`increment=1`):
+
```

Comment:
can you add a screen shot or an image so people visualize what field name are?

Reply:
done